tests/test_API_FB.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from API_FB import Manipulation
+
+class main():
+    # Load our first page
+    search_url = "https://www.facebook.com/careers/jobs/?q="
+
+    # Need to pull from db
+    prefrences = "DevOps"
+    r = requests.get(search_url+prefrences)
```

Comment:
You should have a function that does this, it shouldn't be part of your test code

Reply:
It will be done today
thx